src/abstractdataframe/reshape.jl

```diff
@@ -229,8 +232,12 @@ Row and column keys will be ordered in the order of their first appearance.
   will be created.
 - `allowduplicates`: if `false` (the default) then an error an error will be thrown
   if combination of `rowkeys` and `colkey` contains duplicate entries; if `true`
-  then  then the last encountered `value` will be retained.
-- `fill`: missing row/column combinations are filled with this value. The default
+  then  then the last encountered `value` will be retained; if it is a `Function`
+  then a vector containing all elements for each combination of `rowkeys` and `colkey`
+  will be passed to `allowduplicates` function.
```

Comment:
```suggestion
  then the last encountered `value` will be retained; if it is a `Function`
  then it will be called on a vector containing all elements for each
  combination of `rowkeys` and `colkey`.
```